packages/strapi-parts/src/Field/FieldLabel.js

```diff
@@ -3,16 +3,21 @@ import PropTypes from 'prop-types';
 import { useField } from './FieldContext';
 import { Text } from '../Text';
 
-export const FieldLabel = ({ children, ...props }) => {
+export const FieldLabel = ({ children, required, ...props }) => {
   const { id } = useField();
 
   return (
-    <Text textColor="neutral800" htmlFor={id} small bold as="label" {...props}>
+    <Text textColor="neutral800" htmlFor={id} small bold as="label" required={required} {...props}>
       {children}
+      <Text textColor="danger600">{required && '*'}</Text>
```

Comment:
Woops, could you please put the required around the <Text> instead of inside?

Reply:
sure